src/ucp/wireup/wireup_cm.c

```diff
@@ -987,7 +987,7 @@ ucs_status_t ucp_ep_client_cm_create_uct_ep(ucp_ep_h ucp_ep)
         return status;
     }
 
-    ucp_wireup_ep_set_next_ep(&wireup_ep->super.super, cm_ep);
+    ucp_wireup_ep_set_next_ep(&wireup_ep->super.super, cm_ep, cm_idx);
```

Comment:
`UCP_NULL_RESOURCE`, `cm_idx` is not a transport resource which has to be unprogressed, the same for server below

Reply:
done